### ISSUE TITLE
The OpenSearch engine does not work in Firefox

### ISSUE THREAD

**Issue by txtsd** [lutris/website#561]
An OpenSearch engine was added in #553 but it fails to install in firefox.
Github's [opensearch.xml](https://github.com/opensearch.xml) was used as a reference which works even though it is not served with a `application/opensearchdescription+xml` mimetype.

Here is the spec: https://developer.mozilla.org/en-US/docs/Web/OpenSearch

**Comment by mihawk90** [comment]
The file is being found:
![image](https://user-images.githubusercontent.com/556162/98941146-6af11b80-24ec-11eb-845c-efdd9bc381ba.png)

However it cannot be installed:
```
Firefox could not install the search engine from: https://lutris.net/static/xml/opensearch.92ab651ff074.xml
```

Since you already linked the Documentation, the reason why is given in the Troubleshooting section:

> The xmlns attribute is important — without it you could get the error message "Firefox could not download the search plugin".

It's a different wording, but the same effect.

**Comment by txtsd** [comment]
I can add and submit a PR for that, but I doubt that's going to change anything. Did you read https://github.com/lutris/website/pull/553#issuecomment-707917737?

**Comment by mihawk90** [comment]
Yes, I read the entire PR. Since you were referring to the discovery of the search I wasn't sure if installation was actually tested. But either way it could lead to issues so fixing it according to the documentation doesn't hurt.

**Comment by txtsd** [comment]
I don't get why github's opensearch installs without a hiccup even though it's basically the same.

**Comment by txtsd** [comment]
Does it work now? I don't have access to a desktop to test it atm.

**Comment by strycore** [comment]
Yes, it's working fine now
